### PR TITLE
chore: fixed flow types

### PR DESCRIPTION
**Pull request by pvlv**
It will fix -> https://github.com/dialogs/dialog-web-components/pull/205

### CODE REVIEW

File: src/types.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export interface Range {
 
 export interface Decorator {
   name: string;
-  strategy(text: string): Array<Range>;
+  strategy: (text: string) => Array<Range>;
```
Comment (meafmira) [inline src/types.js]: Why?

Comment (pvlv) [reply src/types.js]: From error:
> property `strategy` is read-only in `Decorator` but writable in object literal in array element > of property`decorators`.

```
export interface Decorator {
  name: string;
  strategy(text: string): Array<Range>; // <- read-only
}
```

```
export interface Decorator {
  name: string;
  strategy: (text: string) => Array<Range>; // <- we use in object literal (emoji decorator and etc)
}
```

Comment (meafmira) [reply src/types.js]: So you should make it not writable in place of error not make it writesble here

File: src/types.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * @flow strict
  */
 
-export type TokenOptions = { [key: string]: mixed };
+export type TokenOptions = { [key: string]: string };
```
Comment (nkt) [inline src/types.js]: value might be `number` for example.

**Pull request merged**